package.json

```diff
@@ -4,7 +4,7 @@
   "version": "2.0.0-beta.3",
   "dependencies": {
     "@types/history": "^4.6.2",
-    "query-string": "6.0.0"
+    "qs": "6.5.1"
```

Comment:
Keep the dependency on query-string but make it a dev dependency, there is a test which previously drops qs in to show that you can use your own serialise / stringify, we should do the opposite now.

Reply:
good idea